server/models/Request.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
+    user_id: {
+        type: Number,
+        required: true,
+    },
+    sitter_id: {
+        type: Number,
+        required: true,
+    },
+    start: {
+        type: Date,
+        required: true,
+    },
+    end: {
+        type: Date,
+        required: true,
+    },
+    accepted: {
+        type: Boolean,
+        default: false,
+    },
+    declined: {
+        type: Boolean,
+        default: false,
+    },
```

Comment:
Do you think it would be better to use a field such as `status` to track all of the possible states - maybe new ones might be added in the future as well? (e.g. `draft`, `submitted`, `cancelled`, `accepted`, `declined`...)

Reply:
yes ugur, it would be better to use status with type string, instead of creating separate fields ...
let me just modify it ...